cvat/apps/documentation/user_guide.md

```diff
@@ -1138,6 +1140,40 @@ Detectors are used to automatically annotate one frame. Supported DL models are
 - This action will automatically annotates one frame.
   In the [Automatic annotation](#automatic-annotation) section you can read how to make automatic annotation of all frames.
 
+## OpenCV
+
+This tools based on Open CV, it is used to create polygons. To work with tool, you need to download it.
```

Comment:
The tool based on [Open CV](https://opencv.org/) Computer Vision library which is an open-source product that includes many CV algorithms. Some of these algorithms can be used to simplify the annotation process.

Reply:
First step to work with OpenCV is to load it into CVAT. Click on the toolbar icon, then click `Load OpenCV`.